docs/docsite/rst/user_guide/windows_dsc.rst

```diff
@@ -84,9 +86,11 @@ options are parameters that are used to define the resource, such as ``Key`` and
 keeping the case as-is is recommended becuase it makes it easier to distinguish DSC
 resource options from Ansible's ``win_dsc`` options.
 
-This is what the Ansible task version of the above DSC Registry resource would look like::
+This is what the Ansible task version of the above DSC Registry resource would look like:
+
+.. code-block:: yaml+jinja
 
-    - name: use win_dsc module with the Registry DSC resource
+    - name: Use win_dsc module with the Registry DSC resource
```

Comment:
I'm just wondering, is there a consensus as to whether we want upper case or lower case in the docs, I treat the name block as a bullet list so don't worry about starting with caps or ending with a sentence but that's just me.

Reply:
Hmm, I find it easier to read though. Especially for descriptions and documentation.

I don't mind that much for integration tests, but anything that ends up in the documentation (user-facing) I prefer to use sensible, pleasing standards.

Reply:
The trouble with using this as justification is that everybody has different preferences and we could get into an endless loop with these types of edits. If @acozine has a standard that she's trying to set with the documentation then we should be following that standard but right now I'm unsure what that would be.

Reply:
In general, Red Hat documentation follows the IBM Style guide. That said, it doesn't cover this scenario directly. Our Ansible style guide also doesn't cover this, but documenting modules has an example section:
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#examples-block

That uses an initial cap. Also the few examples I poked at in https://github.com/ansible/ansible-examples also uses initial cap for names.  I'll ask @acozine directly if she wants to adopt this as a standard (and say so in the docs so it's clear).

Reply:
From a Community point of view we need to be careful not to cause "busy work" for Contributors and reviewers.

I think it's is OK to:

* Update "how to document your module" to state how we'd *like* documentation to be written
* update any examples in `docs/docsite/rst` to use the "best" form (with `name:`, capital letters, etc.)
* Bulk update of `support:core` modules to use the "best" form. 
* If others want to update modules that's fine, though that must be the only change in the PR to make it easy to review and backport. 

I wouldn't want to add review comments on PRs asking them to add capital letters (or fullstops/periods at the end).

Reply:
Chatted offline w/ @acozine and she doesn't have a preference for either way. So I think the 'gist' of it is:

- Don't comment on another PR to request capitalization etc change
- Recommend to always use a name and make it a useful phrase or sentence.
- no bulk changes to existing content, because there's no 'right or wrong' way to do it.